### PR TITLE
Update Microsoft.Resources and Microsoft.Authorization from swagger

### PR DESCRIPTION
**Pull request by daschult**

### CODE REVIEW

**Comment by TianoMS** [comment]
@daschult Left one comment. I also created a PR to add test cases.

**Comment by daschult** [comment]
@TianoMS I don't see your comment, but thanks for creating more tests. :)

File: schemas/2016-09-01/Microsoft.Authorization.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The lock level of the management lock."
+        },
+        "notes": {
+          "type": "string",
+          "description": "The notes of the management lock."
+        },
+        "owners": {
```
Comment (TianoMS) [inline schemas/2016-09-01/Microsoft.Authorization.json]: Yes please do that. We added the property for internal test. We need to generate SDK with the new property.

Comment (daschult) [inline schemas/2016-09-01/Microsoft.Authorization.json]: Removed.

**Comment by TianoMS** [comment]
@daschult Can you merge my PR which adds test cases?

**Comment by daschult** [comment]
@TianoMS I don't see your PR. Can you send me a link?

**Comment by TianoMS** [comment]
@daschult This one: https://github.com/daschult/azure-resource-manager-schemas/pull/4

**Comment by daschult** [comment]
@TianoMS Thanks. That made it much easier. It's merged.

**Pull request merged**